### PR TITLE
Largest Kth Element (DC Problem)

### PR DESCRIPTION
**Pull request by SalmaZakaria**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description
Two sorted arrays A1 and A2 of sizes N and M integers which can be positive, negative, or zero. You want to get the Largest Kth element in the final sorted array.

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#4651

### CODE REVIEW

File: C-Sharp/other/Largest_Kth_Element.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿using System;
```
Comment (HarshCasper) [inline C-Sharp/other/Largest_Kth_Element.cs]: Add a beginner friendly pitch here

File: C-Sharp/other/Largest_Kth_Element.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            return res;
+        }
+        /*
+         * The Problem is finding the largest Kth item in Two Separated sorted Arrays
+         * Complexity O(log(N))
+         */
+        static void Main(string[] args)
+        {
+            int[] A1 = { 1, 2, 3, 4, 5 };
+            int[] A2 = { 7, 9, 15, 17 };
```
Comment (HarshCasper) [inline C-Sharp/other/Largest_Kth_Element.cs]: Use dynamic input

File: C-Sharp/other/Largest_Kth_Element.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            Console.WriteLine("Enter the Kth Item: ");
+            int k = int.Parse(Console.ReadLine());
+            Console.WriteLine("The number is: " + GetKthItem(A1, A2, A1.Length, A2.Length, k)); ;
+        }
+    }
+}
```
Comment (HarshCasper) [inline C-Sharp/other/Largest_Kth_Element.cs:104]: Add sample I/O and complexity

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Sharp/other/Largest_Kth_Element.cs (hunk 1 above)
Comment (SalmaZakaria) [reply C-Sharp/other/Largest_Kth_Element.cs]: @HarshCasper  Done, You can review it again now

File: C-Sharp/other/Largest_Kth_Element.cs (hunk 2 above)
Comment (SalmaZakaria) [reply C-Sharp/other/Largest_Kth_Element.cs]: @HarshCasper  Done, You can review it again now

File: C-Sharp/other/Largest_Kth_Element.cs (hunk 3 above)
Comment (SalmaZakaria) [reply C-Sharp/other/Largest_Kth_Element.cs:104]: @HarshCasper  Done, You can review it again now

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**